dso/model.py

```diff
@@ -721,6 +729,8 @@ def key(self) -> str:
             self.user.key,
             self.comment,
             self.finding.key,
+            str(self.allowed_processing_time),
```

Comment:
hmmm.... I think we should either define the unit (e.g. allowed_processing_time_days), and _not_ convert to str, OR define something more flexible, such as `20d` (for 20 days) - in which case we would not allow int.

or, asking differently: why convert to str here?